db/schema.rb

```diff
@@ -268,9 +268,12 @@
   create_table "broadcasts", id: :serial, force: :cascade do |t|
     t.boolean "active", default: false
     t.text "body_markdown"
+    t.datetime "created_at"
     t.text "processed_html"
     t.string "title"
     t.string "type_of"
+    t.datetime "updated_at"
```

Comment:
I think this PR needs to be rebased on master; looks like [#8181](https://github.com/thepracticaldev/dev.to/pull/8181/files) added the `created_at` and `updated_at` timestamps recently.

Once it's rebased, those two lines shouldn't show up in the `schema.db`, and it should just be the index, as expected :)